dctf2017/archive/chip8-1.1/etc/CHANGES.1.1



Project "chip8.1.1"						  Page 1
List of Changes					Tue Sep 18 13:24:57 2012

Change	State		Description
------- -------		-------------
  10	completed	update build system
  11	completed	fix GNU GPL version
  12	completed	fix author address
  13	completed	improve help
  14	completed	add `low' and `high' super-chip opcodes
  15	completed	construct archive set
  16	completed	fix dif bug
  17	completed	new games from net
  18	completed	add building file
  19	completed	add rest of schip opcodes
  20	completed	add reference manual
  21	completed	ansi insulation
  22	completed	FSF address
  23	completed	add exit schip opcode
  24	completed	more file formats
  25	completed	more chipper opcodes
  26	completed	more chipper opcodes
  27	completed	more chipper opcodes
  28	completed	more chipper opcodes
  29	completed	add game documentation
  30	completed	joust
  31	completed	uboat
  32	completed	hpiper
  33	completed	conditional assembly
  34	completed	blinky
  35	completed	alien
  36	completed	ant
  37	completed	blitz
  38	completed	car
  39	completed	connect4
  40	completed	field
  41	completed	guess
  42	completed	hidden
  43	completed	library path
  44	completed	the invaders game
  45	completed	opcode argument count
  46	completed	the missile game
  47	completed	add kaleid game
  48	completed	hpiper doco
  49	completed	the syzygy game
  50	completed	the maze game
  51	completed	the merlin game
  52	completed	the race game
  53	completed	add html links
  54	completed	the worm3 game
  55	completed	new cook functionality
  56	completed	the space fight game
  57	completed	the tictac game
  58	completed	the vers game
  59	completed	the tank game
  60	completed	the vbrix game
  61	completed	warn of ambiguous i-register behaviour
  62	completed	the 15puzzle game
  63	completed	make it build
  64	completed	fix fsf address



Project "chip8.1.1"						  Page 2
List of Changes					Tue Sep 18 13:24:57 2012

Change	State		Description
------- -------		-------------
  65	completed	remove MANIFEST
  66	completed	web site
  67	completed	fix trailing white space
  68	completed	fix line lengths
  69	completed	modern test infrastructure
  70	completed	aemakegen for tarball
  71	completed	aemakegen for spec file
  72	completed	debian package, part 1
  73	completed	unquoted hyphens
  74	completed	fix pmiller email
  75	completed	fix web site addresses
  76	completed	modern -I methodology
  77	completed	no more K&R definitions
  78	completed	gcc -Wwrite-string
  79	completed	tags
  80	awaiting_	c_incl -excl
	development
  81	completed	libexplain, part 1
  82	awaiting_	quiet build
	development
  83	completed	snprintf
  84	completed	no more K&R, part 2
  85	completed	gcc -Wextra
  86	completed	games => datadir
  87	awaiting_	apt-get-depends.cook
	development
  88	completed	libexplain, part 6, malloc
  89	completed	trace function prototypes
  90	completed	libexplain, part 2, progname
  91	completed	game => datadir, part 2
  92	completed	O(1) string table
  93	completed	small malloc replacements
  94	completed	libexplain, part 4, malloc again
  95	completed	libexplain, part 5, malloc in chip8run
  96	completed	stricter testing mode
  97	completed	debian-package, part 2
  98	completed	assembler listing titles
  99	being_developed debian package, part 3, web site
	pmiller
 100	being_developed test_prelude in package?!?
	pmiller
 101	completed	chip8-licence.1
 102	completed	XX macro
 103	completed	upload-ppa.sh
 104	completed	upload-ppa, again
 105	completed	web site
 106	completed	debian package build depends
 107	completed	debian-description.so
 108	being_		aede-policy strictest
	integrated
	pmiller



Project "chip8.1.1", Change 10					  Page 1
Change Details					Tue Sep 18 13:24:57 2012

NAME
	Project "chip8.1.1", Delta 1, Change 10.

SUMMARY
	update build system

DESCRIPTION
	This change updates the build system.

	This change is exempt from testing against the baseline.


ARCHITECTURE
	This change must build and test in the "linux-i486"
	architecture.

CAUSE
	This change was caused by internal_enhancement.

FILES
	Type	Action	 Edit	      File Name
	------- -------- -------      -----------
	source	remove	    1	      Howto.cook
				      Moved to etc/Howto.cook
	source	modify	    1 -> 2    chip8as/as.c
	source	modify	    1 -> 2    chip8as/as.h
	source	modify	    1 -> 2    chip8as/col.c
	source	modify	    1 -> 2    chip8as/col.h
	source	modify	    1 -> 2    chip8as/emit.c
	source	modify	    1 -> 2    chip8as/emit.h
	source	modify	    1 -> 2    chip8as/id.c
	source	modify	    1 -> 2    chip8as/id.h
	source	modify	    1 -> 2    chip8as/lex.c
	source	modify	    1 -> 2    chip8as/lex.h
	source	modify	    1 -> 2    chip8as/main.c
	source	modify	    1 -> 2    chip8dis/dis.c
	source	modify	    1 -> 2    chip8dis/main.c
	source	modify	    1 -> 2    chip8run/arena.c
	source	modify	    1 -> 2    chip8run/arena.h
	source	modify	    1 -> 2    chip8run/button.c
	source	modify	    1 -> 2    chip8run/button.h
	source	modify	    1 -> 2    chip8run/chip8.c
	source	create	    1	      chip8run/chip8.h
	source	modify	    1 -> 2    chip8run/debug.c
	source	modify	    1 -> 2    chip8run/debug.h
	source	modify	    1 -> 2    chip8run/ecks.c
	source	modify	    1 -> 2    chip8run/ecks.h
	source	modify	    1 -> 2    chip8run/event.c
	source	modify	    1 -> 2    chip8run/event.h
	source	modify	    1 -> 2    chip8run/machine.c
	source	modify	    1 -> 2    chip8run/machine.h
	source	modify	    1 -> 2    chip8run/main.c
	source	modify	    1 -> 2    chip8run/option.c
	source	modify	    1 -> 2    chip8run/shell.c
	source	modify	    1 -> 2    chip8run/shell.h
	source	modify	    1 -> 2    chip8run/tile.c
	source	create	    1	      chip8run/tile.h



Project "chip8.1.1", Change 10					  Page 2
Change Details					Tue Sep 18 13:24:57 2012

	Type	Action	 Edit	      File Name
	------- -------- -------      -----------
	source	modify	    1 -> 2    chip8run/window.c
	source	modify	    1 -> 2    chip8run/window.h
	source	create	    1	      common/ac/unistd.h
	source	modify	    1 -> 2    common/arglex.c
	source	modify	    1 -> 2    common/arglex.h
	source	modify	    1 -> 2    common/error.c
	source	modify	    1 -> 2    common/error.h
	source	modify	    1 -> 2    common/help.c
	source	modify	    1 -> 2    common/help.h
	source	modify	    1 -> 2    common/main.h
	source	modify	    1 -> 2    common/mem.c
	source	modify	    1 -> 2    common/mem.h
	source	modify	    1 -> 2    common/str.c
	source	modify	    1 -> 2    common/str.h
	source	modify	    1 -> 2    common/trace.c
	source	modify	    1 -> 2    common/trace.h
	source	modify	    1 -> 2    common/version.c
	source	modify	    1 -> 2    common/version.h
	source	modify	    1 -> 2    common/word.c
	source	modify	    1 -> 2    common/word.h
	source	modify	    1 -> 2    config
	source	create	    1	      etc/Howto.conf.in
	source	create	    1	      etc/Howto.cook
				      Moved from Howto.cook
	source	create	    1	      etc/autoconf.cook
	source	create	    1	      etc/config.h.body
	source	create	    1	      etc/config.h.head
	source	create	    1	      etc/config.h.tail
	source	create	    1	      etc/configure.in
	source	create	    1	      etc/template/c
	source	create	    1	      etc/template/generic
	source	create	    1	      etc/template/h
	source	create	    1	      etc/template/man
	source	create	    1	      etc/template/sh
	source	create	    1	      etc/template/test
	test	modify	    1 -> 2    test/00/t0001a.sh
	test	modify	    1 -> 2    test/00/t0002a.sh
	test	modify	    1 -> 2    test/00/t0003a.sh
	test	modify	    1 -> 2    test/00/t0004a.sh
	test	modify	    1 -> 2    test/00/t0005a.sh
	test	modify	    1 -> 2    test/00/t0006a.sh
	test	modify	    1 -> 2    test/00/t0007a.sh

HISTORY
	What		When		Who	 Comment
	------		------		-----	 ---------
	new_change	Thu Apr	 9	pmiller
			22:53:15 1998
	develop_begin	Thu Apr	 9	pmiller	 Elapsed time: 2.000
			22:53:21 1998		 days.
	develop_end	Sat Apr 11	pmiller
			09:55:33 1998
	review_pass	Sat Apr 11	pmiller
			09:55:37 1998




Project "chip8.1.1", Change 10					  Page 3
Change Details					Tue Sep 18 13:24:57 2012

	What		When		Who	 Comment
	------		------		-----	 ---------
	integrate_begin Sat Apr 11	pmiller
			09:55:59 1998
	integrate_pass	Sat Apr 11	pmiller
			09:59:44 1998



Project "chip8.1.1", Change 11					  Page 1
Change Details					Tue Sep 18 13:24:58 2012

NAME
	Project "chip8.1.1", Delta 2, Change 11.

SUMMARY
	fix GNU GPL version

DESCRIPTION
	This change updates the GNU GPL to the most recent version.

	This change must pass a full regression test.  This change is
	exempt from testing against the baseline.


ARCHITECTURE
	This change must build and test in the "linux-i486"
	architecture.

CAUSE
	This change was caused by internal_improvement.

FILES
	Type	Action	 Edit	      File Name
	------- -------- -------      -----------
	source	modify	    1 -> 2    MANIFEST.sh
	source	modify	    1 -> 2    Makefile.sh
	source	modify	    2 -> 3    chip8as/as.c
	source	modify	    2 -> 3    chip8as/as.h
	source	modify	    2 -> 3    chip8as/col.c
	source	modify	    2 -> 3    chip8as/col.h
	source	modify	    2 -> 3    chip8as/emit.c
	source	modify	    2 -> 3    chip8as/emit.h
	source	modify	    2 -> 3    chip8as/id.c
	source	modify	    2 -> 3    chip8as/id.h
	source	modify	    2 -> 3    chip8as/lex.c
	source	modify	    2 -> 3    chip8as/lex.h
	source	modify	    2 -> 3    chip8as/main.c
	source	modify	    1 -> 2    chip8as/option.c
	source	modify	    1 -> 2    chip8as/option.h
	source	modify	    2 -> 3    chip8dis/dis.c
	source	modify	    1 -> 2    chip8dis/dis.h
	source	modify	    2 -> 3    chip8dis/main.c
	source	modify	    2 -> 3    chip8run/arena.c
	source	modify	    2 -> 3    chip8run/arena.h
	source	modify	    2 -> 3    chip8run/button.c
	source	modify	    2 -> 3    chip8run/button.h
	source	modify	    2 -> 3    chip8run/chip8.c
	source	modify	    2 -> 3    chip8run/debug.c
	source	modify	    2 -> 3    chip8run/debug.h
	source	modify	    2 -> 3    chip8run/ecks.c
	source	modify	    2 -> 3    chip8run/ecks.h
	source	modify	    2 -> 3    chip8run/event.c
	source	modify	    2 -> 3    chip8run/event.h
	source	modify	    2 -> 3    chip8run/machine.c
	source	modify	    2 -> 3    chip8run/machine.h
	source	modify	    2 -> 3    chip8run/main.c
	source	modify	    2 -> 3    chip8run/option.c
	source	modify	    1 -> 2    chip8run/option.h



Project "chip8.1.1", Change 11					  Page 2
Change Details					Tue Sep 18 13:24:58 2012

	Type	Action	 Edit	      File Name
	------- -------- -------      -----------
	source	modify	    2 -> 3    chip8run/shell.c
	source	modify	    2 -> 3    chip8run/shell.h
	source	modify	    2 -> 3    chip8run/tile.c
	source	modify	    2 -> 3    chip8run/window.c
	source	modify	    2 -> 3    chip8run/window.h
	source	modify	    2 -> 3    common/arglex.c
	source	modify	    2 -> 3    common/arglex.h
	source	modify	    2 -> 3    common/error.c
	source	modify	    2 -> 3    common/error.h
	source	modify	    2 -> 3    common/help.c
	source	modify	    2 -> 3    common/help.h
	source	modify	    2 -> 3    common/main.h
	source	modify	    2 -> 3    common/mem.c
	source	modify	    2 -> 3    common/mem.h
	source	modify	    2 -> 3    common/str.c
	source	modify	    2 -> 3    common/str.h
	source	modify	    2 -> 3    common/trace.c
	source	modify	    2 -> 3    common/trace.h
	source	modify	    2 -> 3    common/version.c
	source	modify	    2 -> 3    common/version.h
	source	modify	    2 -> 3    common/word.c
	source	modify	    2 -> 3    common/word.h
	source	modify	    2 -> 3    config
	source	modify	    1 -> 2    etc/Howto.cook
	source	modify	    1 -> 2    game/brix.asm
	source	modify	    1 -> 2    game/pong.asm
	source	modify	    1 -> 2    game/puzzle.asm
	source	modify	    1 -> 2    game/snake.asm
	source	remove	    1	      h/stdarg.h
	source	remove	    1	      h/stddef.h
	source	remove	    1	      h/stdlib.h
	source	modify	    1 -> 2    man/man1/chip8as.1
	source	modify	    1 -> 2    man/man1/chip8dis.1
	source	modify	    1 -> 2    man/man1/chip8run.1
	source	modify	    1 -> 2    man/man5/chip8as.5
	source	modify	    1 -> 2    man/man5/chip8run.5
	test	modify	    2 -> 3    test/00/t0005a.sh
	test	modify	    2 -> 3    test/00/t0006a.sh
	test	modify	    2 -> 3    test/00/t0007a.sh

HISTORY
	What		When		Who	 Comment
	------		------		-----	 ---------
	new_change	Sat Apr 11	pmiller
			10:07:38 1998
	develop_begin	Sat Apr 11	pmiller	 Elapsed time: 0.073
			10:07:45 1998		 days.
	develop_end	Sat Apr 11	pmiller
			10:40:37 1998
	review_pass	Sat Apr 11	pmiller
			10:40:41 1998
	integrate_begin Sat Apr 11	pmiller
			10:40:44 1998
	integrate_pass	Sat Apr 11	pmiller
			10:43:01 1998



Project "chip8.1.1", Change 12					  Page 1
Change Details					Tue Sep 18 13:24:58 2012

NAME
	Project "chip8.1.1", Delta 3, Change 12.

SUMMARY
	fix author address

DESCRIPTION
	This change updates the author's address, and tightents up the
	man pages, factoring a lot of common information into include
	files.

	This change must pass a full regression test.  This change is
	exempt from testing against the development directory. This
	change is exempt from testing against the baseline.


ARCHITECTURE
	This change must build and test in the "linux-i486"
	architecture.

CAUSE
	This change was caused by internal_improvement.

FILES
	Type	Action	 Edit	      File Name
	------- -------- -------      -----------
	source	modify	    3	      chip8as/main.c
	source	modify	    3	      chip8dis/main.c
	source	modify	    3	      chip8run/main.c
	source	modify	    3	      common/version.c
	source	modify	    2 -> 3    etc/Howto.cook
	source	modify	    1 -> 2    etc/autoconf.cook
	source	modify	    1 -> 2    etc/configure.in
	source	create	    1	      etc/document.cook
	source	modify	    2 -> 3    man/man1/chip8as.1
	source	modify	    2 -> 3    man/man1/chip8dis.1
	source	modify	    2 -> 3    man/man1/chip8run.1
	source	create	    1	      man/man1/z_copyright.so
	source	create	    1	      man/man1/z_exit.so
	source	modify	    2 -> 3    man/man5/chip8as.5
	source	modify	    2 -> 3    man/man5/chip8run.5

HISTORY
	What		When		Who	 Comment
	------		------		-----	 ---------
	new_change	Sat Apr 11	pmiller
			10:44:11 1998
	develop_begin	Sat Apr 11	pmiller	 Elapsed time: 0.510
			10:44:14 1998		 days.
	develop_end	Sat Apr 11	pmiller
			14:33:53 1998
	review_pass	Sat Apr 11	pmiller
			14:33:56 1998
	integrate_begin Sat Apr 11	pmiller
			14:34:04 1998
	integrate_pass	Sat Apr 11	pmiller
			14:36:23 1998



Project "chip8.1.1", Change 13					  Page 1
Change Details					Tue Sep 18 13:24:58 2012

NAME
	Project "chip8.1.1", Delta 4, Change 13.

SUMMARY
	improve help

DESCRIPTION
	This change improves the integration (mostly updating) of the -
	help options.

	This change must pass a full regression test.  This change is
	exempt from testing against the development directory. This
	change is exempt from testing against the baseline.


ARCHITECTURE
	This change must build and test in the "linux-i486"
	architecture.

CAUSE
	This change was caused by internal_improvement.

FILES
	Type	Action	 Edit	      File Name
	------- -------- -------      -----------
	source	modify	    3 -> 4    chip8as/main.c
	source	modify	    3 -> 4    chip8dis/main.c
	source	modify	    3 -> 4    chip8run/main.c
	source	modify	    3 -> 4    common/arglex.c
	source	modify	    3 -> 4    common/arglex.h
	source	modify	    3 -> 4    common/help.c
	source	modify	    3 -> 4    common/help.h
	source	create	    1	      common/versio_stamp.c
	source	create	    1	      common/versio_stamp.h
	source	modify	    3 -> 4    common/version.c
	source	modify	    3 -> 4    common/version.h
	source	modify	    3 -> 4    etc/Howto.cook
	source	create	    1	      man/man1/chip8lic.1

HISTORY
	What		When		Who	 Comment
	------		------		-----	 ---------
	new_change	Sat Apr 11	pmiller
			14:46:47 1998
	develop_begin	Sat Apr 11	pmiller	 Elapsed time: 0.304
			14:46:50 1998		 days.
	develop_end	Sat Apr 11	pmiller
			17:03:28 1998
	review_pass	Sat Apr 11	pmiller
			17:04:00 1998
	integrate_begin Sat Apr 11	pmiller
			17:04:03 1998
	integrate_pass	Sat Apr 11	pmiller
			17:05:02 1998



Project "chip8.1.1", Change 14					  Page 1
Change Details					Tue Sep 18 13:24:58 2012

NAME
	Project "chip8.1.1", Delta 8, Change 14.

SUMMARY
	add `low' and `high' super-chip opcodes

DESCRIPTION
	This change adds the `low' and `high' opcodes from the new
	Super-Chip spec.  This also adds the burden of supporting and
	maintining the larger display resolution.

	This change is exempt from testing against the baseline.


ARCHITECTURE
	This change must build and test in the "linux-i486"
	architecture.

CAUSE
	This change was caused by internal_enhancement.

FILES
	Type	Action	 Edit	      File Name
	------- -------- -------      -----------
	source	modify	    4 -> 5    chip8as/as.c
	source	modify	    3 -> 4    chip8as/emit.c
	source	modify	    3 -> 4    chip8dis/dis.c
	source	modify	    3 -> 4    chip8run/arena.c
	source	modify	    3 -> 4    chip8run/arena.h
	source	modify	    4 -> 5    chip8run/machine.c
	source	modify	    3 -> 4    chip8run/machine.h
	source	modify	    1 -> 2    etc/archive.cook
	source	modify	    1 -> 2    etc/document.cook
	source	modify	    1 -> 2    man/man1/z_copyright.so
	source	modify	    3 -> 4    man/man5/chip8as.5
	test	modify	    3 -> 4    test/00/t0005a.sh
	test	modify	    3 -> 4    test/00/t0006a.sh
	test	create	    1	      test/00/t0011a.sh

HISTORY
	What		When		Who	 Comment
	------		------		-----	 ---------
	new_change	Sat Apr 11	pmiller
			16:21:14 1998
	develop_begin	Sat Apr 11	pmiller	 Elapsed time: 2.000
			16:21:20 1998		 days.
	develop_end	Tue Apr 14	pmiller
			07:17:45 1998
	review_pass	Tue Apr 14	pmiller
			07:17:47 1998
	integrate_begin Tue Apr 14	pmiller
			07:17:54 1998
	integrate_pass	Tue Apr 14	pmiller
			07:19:47 1998



Project "chip8.1.1", Change 15					  Page 1
Change Details					Tue Sep 18 13:24:58 2012

NAME
	Project "chip8.1.1", Delta 5, Change 15.

SUMMARY
	construct archive set

DESCRIPTION
	This change constructs the archive set, for upload.

	This change must pass a full regression test.  This change is
	exempt from testing against the development directory. This
	change is exempt from testing against the baseline.


ARCHITECTURE
	This change must build and test in the "linux-i486"
	architecture.

CAUSE
	This change was caused by internal_enhancement.

FILES
	Type	Action	 Edit	      File Name
	------- -------- -------      -----------
	source	remove	    1	      LICENSE
	source	remove	    1	      README
				      Moved to etc/README.man
	source	create	    1	      etc/BUILDING.man
	source	create	    1	      etc/CHANGES.1.0
	source	create	    1	      etc/CHANGES.sh
	source	modify	    4 -> 5    etc/Howto.cook
	source	create	    1	      etc/MANIFEST.sh
	source	create	    1	      etc/Makefi.file.sh
	source	create	    1	      etc/Makefile.awk
	source	create	    1	      etc/Makefile.head
	source	create	    1	      etc/Makefile.sh
	source	create	    1	      etc/README.man
				      Moved from README
	source	create	    1	      etc/archive.cook
	source	create	    1	      etc/chip8.gif.uue
	source	create	    1	      etc/chip8.html
	source	create	    1	      etc/chip8.lsm
	source	modify	    2 -> 3    etc/configure.in
	source	create	    1	      etc/rpm-build.sh
	source	create	    1	      etc/spec.sh
	source	create	    1	      etc/ssp.awk

HISTORY
	What		When		Who	 Comment
	------		------		-----	 ---------
	new_change	Sat Apr 11	pmiller
			17:06:18 1998
	develop_begin	Sat Apr 11	pmiller	 Elapsed time: 0.937
			17:06:21 1998		 days.
	develop_end	Sun Apr 12	pmiller
			00:08:08 1998




Project "chip8.1.1", Change 15					  Page 2
Change Details					Tue Sep 18 13:24:58 2012

	What		When		Who	 Comment
	------		------		-----	 ---------
	review_pass	Sun Apr 12	pmiller
			00:08:19 1998
	integrate_begin Sun Apr 12	pmiller
			00:08:22 1998
	integrate_pass	Sun Apr 12	pmiller
			00:12:12 1998



Project "chip8.1.1", Change 16					  Page 1
Change Details					Tue Sep 18 13:24:59 2012

NAME
	Project "chip8.1.1", Delta 6, Change 16.

SUMMARY
	fix dif bug

DESCRIPTION
	This fixes a bug when assembling dif opcodes.


ARCHITECTURE
	This change must build and test in the "linux-i486"
	architecture.

CAUSE
	This change was caused by internal_bug.

FILES
	Type	Action	 Edit	      File Name
	------- -------- -------      -----------
	source	modify	    3 -> 4    chip8as/as.c
	source	modify	    3 -> 4    chip8run/machine.c
	source	modify	    2 -> 3    game/brix.asm
	source	modify	    2 -> 3    game/pong.asm
	source	modify	    2 -> 3    game/snake.asm
	test	create	    1	      test/00/t0008a.sh
	test	create	    1	      test/00/t0009a.sh
	test	create	    1	      test/00/t0010a.sh

HISTORY
	What		When		Who	 Comment
	------		------		-----	 ---------
	new_change	Sun Apr 12	pmiller
			19:06:35 1998
	develop_begin	Sun Apr 12	pmiller	 Elapsed time: 0.550
			19:06:38 1998		 days.
	develop_end	Sun Apr 12	pmiller
			23:14:21 1998
	review_pass	Sun Apr 12	pmiller
			23:14:25 1998
	integrate_begin Sun Apr 12	pmiller
			23:14:30 1998
	integrate_pass	Sun Apr 12	pmiller
			23:16:22 1998



Project "chip8.1.1", Change 17					  Page 1
Change Details					Tue Sep 18 13:24:59 2012

NAME
	Project "chip8.1.1", Delta 7, Change 17.

SUMMARY
	new games from net

DESCRIPTION
	This change adds new games from the net.

	This change must pass a full regression test.  This change is
	exempt from testing against the development directory. This
	change is exempt from testing against the baseline.


ARCHITECTURE
	This change must build and test in the "linux-i486"
	architecture.

CAUSE
	This change was caused by internal_improvement.

FILES
	Type	Action	 Edit	      File Name
	------- -------- -------      -----------
	source	modify	    3 -> 4    config
	source	create	    1	      etc/template/asm
	source	create	    1	      game/titris.asm
	source	create	    1	      game/ufo.asm
	source	create	    1	      game/wipeoff.asm

HISTORY
	What		When		Who	 Comment
	------		------		-----	 ---------
	new_change	Mon Apr 13	pmiller
			09:20:31 1998
	develop_begin	Mon Apr 13	pmiller	 Elapsed time: 0.041
			09:20:45 1998		 days.
	develop_end	Mon Apr 13	pmiller
			09:39:10 1998
	review_pass	Mon Apr 13	pmiller
			09:39:17 1998
	integrate_begin Mon Apr 13	pmiller
			09:39:19 1998
	integrate_pass	Mon Apr 13	pmiller
			09:40:21 1998



Project "chip8.1.1", Change 18					  Page 1
Change Details					Tue Sep 18 13:24:59 2012

NAME
	Project "chip8.1.1", Delta 11, Change 18.

SUMMARY
	add building file

DESCRIPTION
	This change adds the contents of the BUILDING file.

	This change is exempt from testing against the development
	directory. This change is exempt from testing against the
	baseline.


ARCHITECTURE
	This change must build and test in the "linux-i486"
	architecture.

CAUSE
	This change was caused by internal_bug.

FILES
	Type	Action	 Edit	      File Name
	------- -------- -------      -----------
	source	modify	    1 -> 2    etc/BUILDING.man
	source	modify	    7 -> 8    etc/Howto.cook

HISTORY
	What		When		Who	 Comment
	------		------		-----	 ---------
	new_change	Tue Apr 14	pmiller
			07:20:50 1998
	develop_begin	Tue Apr 14	pmiller	 Elapsed time: 3.000
			07:21:29 1998		 days.
	develop_end	Thu Apr 16	pmiller
			23:20:09 1998
	review_pass	Thu Apr 16	pmiller
			23:20:13 1998
	integrate_begin Thu Apr 16	pmiller
			23:20:16 1998
	integrate_pass	Thu Apr 16	pmiller
			23:21:53 1998



Project "chip8.1.1", Change 19					  Page 1
Change Details					Tue Sep 18 13:24:59 2012

NAME
	Project "chip8.1.1", Delta 10, Change 19.

SUMMARY
	add rest of schip opcodes

DESCRIPTION
	This change adds the rest of the Super-Chip opcodes.


ARCHITECTURE
	This change must build and test in the "linux-i486"
	architecture.

CAUSE
	This change was caused by internal_enhancement.

FILES
	Type	Action	 Edit	      File Name
	------- -------- -------      -----------
	source	modify	    5 -> 6    chip8as/as.c
	source	modify	    5 -> 6    chip8as/emit.c
	source	modify	    4 -> 5    chip8as/lex.c
	source	modify	    3 -> 4    chip8as/lex.h
	source	modify	    5 -> 6    chip8as/main.c
	source	modify	    2 -> 3    chip8as/option.c
	source	modify	    2 -> 3    chip8as/option.h
	source	create	    1	      chip8as/output/c.c
	source	create	    1	      chip8as/output/c.h
	source	create	    1	      chip8as/output/hp.c
	source	create	    1	      chip8as/output/hp.h
	source	create	    1	      chip8as/output/raw.c
	source	create	    1	      chip8as/output/raw.h
	source	create	    1	      chip8as/output/unix.c
	source	create	    1	      chip8as/output/unix.h
	source	modify	    5 -> 6    chip8dis/dis.c
	source	create	    1	      chip8run/hex10.asm
	source	create	    1	      chip8run/hex5.asm
	source	modify	    6 -> 7    chip8run/machine.c
	source	modify	    4 -> 5    chip8run/machine.h
	source	modify	    6 -> 7    etc/Howto.cook
	source	modify	    1 -> 2    etc/Makefi.file.sh
	source	modify	    4 -> 5    man/man5/chip8as.5
	test	create	    1	      test/00/t0012a.sh
	test	create	    1	      test/00/t0013a.sh
	test	create	    1	      test/00/t0014a.sh
	test	create	    1	      test/00/t0015a.sh
	test	create	    1	      test/00/t0016a.sh

HISTORY
	What		When		Who	 Comment
	------		------		-----	 ---------
	new_change	Tue Apr 14	pmiller
			07:21:25 1998
	develop_begin	Tue Apr 14	pmiller	 Elapsed time: 3.000
			07:21:31 1998		 days.




Project "chip8.1.1", Change 19					  Page 2
Change Details					Tue Sep 18 13:24:59 2012

	What		When		Who	 Comment
	------		------		-----	 ---------
	develop_end	Thu Apr 16	pmiller
			17:17:59 1998
	review_pass	Thu Apr 16	pmiller
			17:18:03 1998
	integrate_begin Thu Apr 16	pmiller
			17:18:06 1998
	integrate_pass	Thu Apr 16	pmiller
			17:20:34 1998



Project "chip8.1.1", Change 20					  Page 1
Change Details					Tue Sep 18 13:24:59 2012

NAME
	Project "chip8.1.1", Delta 12, Change 20.

SUMMARY
	add reference manual

DESCRIPTION
	This change addsa a reference manual, combining all the man
	pages, plus some explanatory blurb.

	This change must pass a full regression test.  This change is
	exempt from testing against the development directory. This
	change is exempt from testing against the baseline.


ARCHITECTURE
	This change must build and test in the "linux-i486"
	architecture.

CAUSE
	This change was caused by internal_enhancement.

FILES
	Type	Action	 Edit	      File Name
	------- -------- -------      -----------
	source	modify	    8 -> 9    etc/Howto.cook
	source	modify	    2 -> 3    etc/archive.cook
	source	modify	    2 -> 3    etc/document.cook
	source	create	    1	      etc/ref-ptx.ignore
	source	create	    1	      etc/ref-ptx.sh
	source	create	    1	      etc/ref-ptx1.awk
	source	create	    1	      etc/ref-ptx2.awk
	source	create	    1	      etc/ref-toc.so
	source	create	    1	      etc/reference.man
	source	modify	    3 -> 4    man/man1/chip8as.1
	source	modify	    3 -> 4    man/man1/chip8dis.1
	source	modify	    1 -> 2    man/man1/chip8lic.1
	source	modify	    3 -> 4    man/man1/chip8run.1
	source	modify	    5 -> 6    man/man5/chip8as.5
	source	modify	    3 -> 4    man/man5/chip8run.5

HISTORY
	What		When		Who	 Comment
	------		------		-----	 ---------
	new_change	Tue Apr 14	pmiller
			07:22:21 1998
	develop_begin	Tue Apr 14	pmiller	 Elapsed time: 3.343
			07:22:32 1998		 days.
	develop_end	Fri Apr 17	pmiller
			09:56:46 1998
	review_pass	Fri Apr 17	pmiller
			09:56:56 1998
	integrate_begin Fri Apr 17	pmiller
			09:56:58 1998
	integrate_pass	Fri Apr 17	pmiller
			09:59:11 1998



Project "chip8.1.1", Change 21					  Page 1
Change Details					Tue Sep 18 13:24:59 2012

NAME
	Project "chip8.1.1", Delta 9, Change 21.

SUMMARY
	ansi insulation

DESCRIPTION
	This change adds the ansi insulation.

	This change must pass a full regression test.  This change is
	exempt from testing against the development directory. This
	change is exempt from testing against the baseline.


ARCHITECTURE
	This change must build and test in the "linux-i486"
	architecture.

CAUSE
	This change was caused by internal_improvement.

FILES
	Type	Action	 Edit	      File Name
	------- -------- -------      -----------
	source	modify	    3 -> 4    chip8as/col.c
	source	modify	    4 -> 5    chip8as/emit.c
	source	modify	    3 -> 4    chip8as/id.c
	source	modify	    3 -> 4    chip8as/lex.c
	source	modify	    4 -> 5    chip8as/main.c
	source	modify	    4 -> 5    chip8dis/dis.c
	source	modify	    4 -> 5    chip8dis/main.c
	source	modify	    4 -> 5    chip8run/arena.c
	source	modify	    3 -> 4    chip8run/button.c
	source	modify	    3 -> 4    chip8run/debug.c
	source	modify	    3 -> 4    chip8run/ecks.c
	source	modify	    5 -> 6    chip8run/machine.c
	source	modify	    4 -> 5    chip8run/main.c
	source	modify	    3 -> 4    chip8run/shell.c
	source	modify	    3 -> 4    chip8run/window.c
	source	create	    1	      common/ac/ctype.h
	source	create	    1	      common/ac/fcntl.h
	source	create	    1	      common/ac/limits.h
	source	create	    1	      common/ac/stdarg.h
	source	create	    1	      common/ac/stddef.h
	source	create	    1	      common/ac/stdio.h
	source	create	    1	      common/ac/stdlib.h
	source	create	    1	      common/ac/string.c
	source	create	    1	      common/ac/string.h
	source	create	    1	      common/ac/time.h
	source	modify	    4 -> 5    common/arglex.c
	source	modify	    3 -> 4    common/error.c
	source	modify	    3 -> 4    common/mem.c
	source	modify	    3 -> 4    common/mem.h
	source	remove	    1	      common/patchlevel.h
	source	modify	    3 -> 4    common/str.c
	source	modify	    3 -> 4    common/str.h
	source	modify	    3 -> 4    common/trace.c



Project "chip8.1.1", Change 21					  Page 2
Change Details					Tue Sep 18 13:24:59 2012

	Type	Action	 Edit	      File Name

dctf2017/archive/chip8-1.1/etc/CHANGES.sh
#!/bin/sh
#
#	chip8 - X11 Chip8 interpreter
#	Copyright (C) 1998, 2012 Peter Miller
#
#	This program is free software; you can redistribute it and/or modify
#	it under the terms of the GNU General Public License as published by
#	the Free Software Foundation; either version 2 of the License, or
#	(at your option) any later version.
#
#	This program is distributed in the hope that it will be useful,
#	but WITHOUT ANY WARRANTY; without even the implied warranty of
#	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
#	GNU General Public License for more details.
#
#	You should have received a copy of the GNU General Public License
#	along with this program. If not, see
#	<http://www.gnu.org/licenses/>.
#
case $# in
1)
	project=$1
	;;

*)
	echo "usage: $0 <project>" 1>&2
	exit 1
	;;
esac
set -e
aegis -l changes -p $project -v -pl 66 -pw 72 < /dev/null
changes=`aegis -list changes -terse -project $project`
for n in $changes
do
	echo "" | tr '\12' '\14'
	aegis -l change_details -p $project -c $n -v -pl 66 -pw 72 < /dev/null
done
exit 0
